internal/host/transition.go

```diff
@@ -71,6 +71,12 @@ var restFieldsOnUnbind = append(append(resetProgressFields, resetLogsField...),
 	"role", "auto-assign", "api_vip_connectivity", "", "suggested_role", "", "images_status", "",
 	"stage_started_at", strfmt.DateTime(time.Time{}), "stage_updated_at", strfmt.DateTime(time.Time{}))
 
+var defaultPendingUserActionErrorFormat = "Host is required to be booted from disk %s"
+var platformPendingUserActionErrorFormat = map[string]string{
+	"nutanix": "Unmount CDROM in VM properties to be booted from disk %s",
```

Comment:
i dont think this is what we should tell them to do since it will work once. we should guide them how to set it right - we already do it with libvirt somewhere

Reply:
We guide them how to do that correctly in the docs, but people make mistakes and the error message should be clear and have a suggestion on how to fix the situation